package.json

```diff
@@ -114,7 +115,7 @@
     "@babel/preset-typescript": "^7.16.7",
     "@edx/eslint-config": "^3.2.0",
     "@edx/stylelint-config-edx": "^2.3.0",
-    "@edx/typescript-config": "^1.0.1",
+    "@edx/typescript-config": "^1.1.0",
```

Comment:
Just curious: is this change necessary for this PR? No doubt it's a *good* change, just wondering if was a nice-to-have or a need-to-have.